builder/steps/gen-dockerfile/contents/src/detect_setup.ts

```diff
@@ -123,35 +129,40 @@ export async function detectSetup(
 
   logger.log('Checking for Node.js.');
 
+  // Consider a file as present if and only if the file exists and is not
+  // specified as being skipped in the deploment yaml file.
+  let skipFiles = config.skip_files || [];
+  if (!Array.isArray(skipFiles)) {
+    skipFiles = [skipFiles];
+  }
+
+  function isSkipped(filename: string): boolean {
+    return skipFiles.some(
+        (pattern: string) => new RegExp(pattern).test(filename));
```

Comment:
nit: I would find it more readable if there were curly braces around the function body.

Reply:
Fixed